### PR TITLE
FAQ: Add Graphene Enclave Manager ZMQ socket URL

### PR DESCRIPTION
**Pull request by danintel**
Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

**Comment by danintel** [comment]
Related to PR #612

File: docs/FAQ.rst [hunk 1]
```diff
@@ -265,9 +265,15 @@ What TCP ports does Avalon use?
   The URL is ``http://localhost:9090/`` or, for Docker,
   ``http://avalon-lmdb:9090/``
 - TCP 5555: ZMQ connections to Avalon Enclave Manager from Avalon Listener.
-  This is used for Avalon singleton enclave workers using Synchronous Mode
+  This is used for Avalon singleton enclave workers using Synchronous Mode.
   The URL is ``tcp://localhost:5555`` or, for Docker,
   ``tcp://avalon-enclave-manager:5555``
+- TCP 7777: ZMQ connections to Avalon Graphene Enclave Manager from
+  Avalon Listener.
```
Comment (manojgop) [inline docs/FAQ.rst:272]: ZMQ socket port used by Avalon Graphene Enclave Manager to communicate with Graphene Python Worker.

File: docs/FAQ.rst [hunk 2]
```diff
@@ -265,9 +265,15 @@ What TCP ports does Avalon use?
...
+  This is used for Avalon's Graphene Enclave Manager to listen to 
+  requests through a ZMQ socket.
```
Comment (manojgop) [inline docs/FAQ.rst:274]: This port is used by Avalon's Graphene Enclave Manager to send request to Graphene Python Worker and receive the response.

File: docs/FAQ.rst [hunk 3]
```diff
@@ -265,9 +265,15 @@ What TCP ports does Avalon use?
...
+  The URL is ``tcp://localhost:7777`` or, for Docker,
+  ``tcp://process-work-order:7777``
```
Comment (manojgop) [inline docs/FAQ.rst:276]: Currently only docker mode is supported for Graphene Python worker. So do we need to mention tcp://localhost:7777 ?

File: docs/FAQ.rst (hunk 1 above)
Comment (danintel) [reply docs/FAQ.rst:272]: @manojgop Thanks for the correction.

File: docs/FAQ.rst [hunk 4]
```diff
@@ -265,9 +265,13 @@ What TCP ports does Avalon use?
   The URL is ``http://localhost:9090/`` or, for Docker,
   ``http://avalon-lmdb:9090/``
 - TCP 5555: ZMQ connections to Avalon Enclave Manager from Avalon Listener.
-  This is used for Avalon singleton enclave workers using Synchronous Mode
+  This is used for Avalon singleton enclave workers using Synchronous Mode.
```
Comment (rranjan3) [inline docs/FAQ.rst:268]: used for --> used by

File: docs/FAQ.rst (hunk 3 above)
Comment (manojgop) [reply docs/FAQ.rst:274]: For docker it's `tcp:://graphene-python-worker:7777`

**Pull request merged**